Task40/src/components/inputPlus.js

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { increment } from "../redux/todos/todoSlice";
+
+export const InputPlus = () => {
```

Comment:
```suggestion
export const AddTodoForm = () => {
```
файл теж перейменуйте